src/main/frontend/pipeline-console-view/pipeline-console/main/stage-details.tsx

```diff
@@ -80,7 +80,7 @@ export default function StageDetails({ stage }: StageDetailsProps) {
           </Tooltip>
         </li>
         {stage.pauseDurationMillis !== 0 && (
-          <li>
+          <li className={"jenkins-mobile-hide"}>
```

Comment:
one thing to keep in mind here is that nested stages don't show in the graph so its not possible to navigate on mobile then. I wonder if it would be better to move the tree to the top in some way on mobile?

Reply:
Definitely something to look into I think - still worth getting this in though as currently its a mess on lower resolutions.